compiler/rustc_session/messages.ftl

```diff
@@ -24,6 +24,9 @@ session_feature_diagnostic_for_issue =
 session_feature_diagnostic_help =
     add `#![feature({$feature})]` to the crate attributes to enable
 
+session_feature_suggest_upgrade_compiler =
+    this compiler is version {$version} built on {$date}, consider upgrading?
```

Comment:
The "consider upgrading?" seems awkward here, especially if we are not going to make this conditional.

```suggestion
    this compiler was built on {$date}; consider upgrading to the latest version if it is out of date
```

Reply:
I have applied this with a small tweak, I left out the "to the latest version" bit, because it may be possible to upgrade to newer but not latest version and fix the issue described in the diagnostic